examples/material-ui-nextjs-app-router-tailwind-ts/src/components/mui-provider.tsx

```diff
@@ -0,0 +1,23 @@
+'use client';
+
+import * as React from 'react';
+import { ThemeProvider } from '@mui/material/styles';
+import CssBaseline from '@mui/material/CssBaseline';
+
+import NextAppDirEmotionCacheProvider from '@/components/emotion-cache';
+import theme from '@/lib/theme';
+
+export type MuiProviderProps = {
+  children?: React.ReactNode;
+};
+
+export function MuiProvider({ children }: MuiProviderProps) {
+  return (
+    <NextAppDirEmotionCacheProvider options={{ key: 'mui' }}>
```

Comment:
Please switch to use https://mui.com/material-ui/integrations/nextjs/#configuration